apps/portfolio/src/app/features/blog/blog-list/blog-list.component.scss

```
@@ -382,32 +317,18 @@
 .blog-hero {
   display: grid;
   grid-template-columns: 1fr;
-  
-  // Custom futuristic background gradient & variables
-  --card-bg: linear-gradient(135deg, rgba(16, 20, 28, 0.82) 0%, rgba(9, 11, 16, 0.96) 100%);
-  --card-grid-color: color-mix(in srgb, var(--card-glow-color, var(--accent-primary)) 12%, transparent);
-  --card-grid-opacity: 0.65;
-  
-  background: radial-gradient(circle at bottom right, color-mix(in srgb, var(--card-glow-color, var(--accent-primary)) 3%, transparent), transparent 60%), var(--card-bg);
+
+  background: var(--surface-primary);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)